tests/compass-runtime-agent/test/testkit/assertions/k8s_resource_checker.go

```diff
@@ -210,6 +210,12 @@ func (c *K8sResourceChecker) assertAssetGroup(t *testing.T, apiPackage *graphql.
 			source, found := getAPISource(api.ID, assetGroup)
 			require.True(t, found)
 
+			apiDescription := ""
+			if api.Description != nil {
+				apiDescription = *api.Description
+			}
+			assert.Equal(t, apiDescription, source.DisplayName)
```

Comment:
I'm not sure if above comments yield additional changes here...

Reply:
Done